src/xb-database-manager.c

```diff
@@ -39,6 +39,7 @@
 #define STOPWORDS_METADATA_KEY "XbStopwords"
 #define QUERY_PARSER_FLAGS XAPIAN_QUERY_PARSER_FEATURE_DEFAULT | \
   XAPIAN_QUERY_PARSER_FEATURE_WILDCARD |\
+  XAPIAN_QUERY_PARSER_FEATURE_PURE_NOT |\
```

Comment:
If we enable this right now in master, do we need code in the existing engine to check that the passed-in query string does not contain a "NOT" statement? I want to make sure that we are in full control of what goes flagged as NOT in the query.